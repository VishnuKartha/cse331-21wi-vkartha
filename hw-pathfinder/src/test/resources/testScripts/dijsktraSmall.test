# Find a path with Dijkstra's algorithm with a small graph

CreateGraph smallGraph

AddNode smallGraph A
AddNode smallGraph B
AddNode smallGraph C


AddEdge smallGraph A B 2.0
AddEdge smallGraph B C 1.0
AddEdge smallGraph A C 10.0



FindPath smallGraph A C
