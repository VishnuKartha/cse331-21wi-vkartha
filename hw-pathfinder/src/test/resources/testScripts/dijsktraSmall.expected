# Find a path with Dijkstra's algorithm with a small graph

created graph smallGraph

added node A to smallGraph
added node B to smallGraph
added node C to smallGraph


added edge 2.000 from A to B in smallGraph
added edge 1.000 from B to C in smallGraph
added edge 10.000 from A to C in smallGraph



path from A to C:
A to B with weight 2.000
B to C with weight 1.000
total cost: 3.000